skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerConfig.scala

```diff
@@ -31,6 +31,7 @@ final case class ConsumerConfig(
   excludeInternalTopics: Boolean             = true,
   isolationLevel: IsolationLevel             = IsolationLevel.ReadUncommitted,
   saslSupport: SaslSupportConfig             = SaslSupportConfig.Default,
+  sslSupport: SslSupportConfig               = SslSupportConfig.Default,
```

Comment:
this is not backward compatible change, could you please use same approach as here?
https://github.com/evolution-gaming/scassandra/blob/master/scassandra/src/main/scala/com/evolutiongaming/scassandra/CassandraConfig.scala#L37